### ISSUE TITLE
【功能请求】优化输出 计算平均延迟结果？比如丢包率/延迟？

### ISSUE THREAD

**Issue by lirener** [kyan001/ping3#96]
C:\Users\Administrator\Desktop>fping 192.168.2.2 -l
192.168.2.2 : [0], 84 bytes, 0.62 ms (0.62 avg, 0% loss)
192.168.2.2 : [1], 84 bytes, 0.48 ms (0.55 avg, 0% loss)
192.168.2.2 : [2], 84 bytes, 0.47 ms (0.52 avg, 0% loss)
192.168.2.2 : [3], 84 bytes, 0.48 ms (0.51 avg, 0% loss)

192.168.2.2 : xmt/rcv/%loss = 4/4/0%, min/avg/max = 0.47/0.51/0.62

-----------

C:\Users\Administrator\Desktop>ping3 192.168.2.2
ping '192.168.2.2' ... 0ms
ping '192.168.2.2' ... 0ms
ping '192.168.2.2' ... 0ms
ping '192.168.2.2' ... 0ms

我想用ping3做网络故障和设备检测 希望他在持续参数中 有统计的功能
fping不支持ipv6，ping3支持ipv6但目前不支持统计。

**Comment by lirener** [comment]
~~我的网络监控项目：https://github.com/game-turn-over-skill-group/Network-Monitor~~
~~我想接入ping3做 网络故障检测~~（2026年05月29日 18:35:38 已经加入完毕：支持icmping+tcping_log）

<img width="2546" height="686" alt="Image" src="https://github.com/user-attachments/assets/a62e7453-c857-4ba5-9f0e-3e0a14cabf37" />

~~我的udping项目（它支持学习了fping的统计）可以拿来做参考？：~~
~~https://github.com/game-turn-over-skill-group/udping~~
